src/core/jpg.js

```diff
@@ -235,7 +235,8 @@ var JpegImage = (function JpegImageClosure() {
       var s;
       var rs;
       while (k <= e) {
-        var z = dctZigZag[k];
+        let offsetZ = offset + dctZigZag[k];
+        let sign = component.blockData[offsetZ] < 0 ? -1 : 1;
```

Comment:
I suppose that we could have re-added `Util.sign`, i.e. revert commit https://github.com/mozilla/pdf.js/commit/5b67c7594cce0ff48e73420f2a1d08b12e3da0ba, but that just didn't seem worthwhile given that this is only used in *one single* spot in the code.